src/beautifiers/tidy-markdown.coffee

```diff
@@ -1,16 +1,26 @@
 "use strict"
 Beautifier = require('./beautifier')
+beautifyCodeBlocks = require './markdown-blocks'
 
 module.exports = class TidyMarkdown extends Beautifier
   name: "Tidy Markdown"
   link: "https://github.com/slang800/tidy-markdown"
   options: {
-    Markdown: false
+    _: {
+      beautifyCodeBlocks: true
+    }
+    Markdown: true
   }
 
   beautify: (text, language, options) ->
+    logger = @logger
     return new @Promise((resolve, reject) ->
       tidyMarkdown = require 'tidy-markdown'
-      cleanMarkdown = tidyMarkdown(text)
-      resolve(cleanMarkdown)
+      cleanMarkdown = tidyMarkdown(text, logger)
```

Comment:
I don't think you meant to pass the `logger` to `tidyMarkdown`: https://github.com/slang800/tidy-markdown/blob/master/src/index.coffee#L234

Maybe to `beautifyCodeBlocks` instead?